system-tests/services/test-apps.js

```diff
@@ -92,6 +110,13 @@ describe("Services", function() {
       // Run service
       cy.get("button.button-primary").contains("Run Service").click();
 
+      // Also no Error should exist
+      cy
+        .wait("@appsReq")
+        .wait(1000)
```

Comment:
without that second wait() cypress tries to get `.alert-danger` before the page is loaded/rendered and then assumes everything is fine. 
But TBH I'm not sure anymore if this check is needed at all. if there is an error, the next check will fail anyway.